CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,14 @@
 - Add: travis.yml file for Travis CI. All branches with Java 8,9,10,11.
+- Upgrade org.slf4j (api & log4j) from 1.6.1 to 12 1.6.1
```

Comment:
"...from 1.6.1 to 12 1.6.1"

Typo?

Reply:
Fixed.
And it was also necessary to add the maven-war-plugin upgrade